repo2docker/buildpacks/nix/__init__.py

```diff
@@ -23,17 +23,20 @@ def get_build_scripts(self):
         """
         Return series of build-steps common to all nix repositories.
         Notice how only root privileges are needed for creating nix
-        directory.
+        directory and a nix.conf file.
 
          - create nix directory for user nix installation
+         - disable sandboxing because its unsupported inside a Docker container
          - install nix package manager for user
+
         """
         return super().get_build_scripts() + [
             (
                 "root",
                 """
             mkdir -m 0755 /nix && \
-            chown -R ${NB_USER}:${NB_USER} /nix /usr/local/bin/nix-shell-wrapper /home/${NB_USER}
+            chown -R ${NB_USER}:${NB_USER} /nix /usr/local/bin/nix-shell-wrapper /home/${NB_USER} && \
+            mkdir -p /etc/nix && touch /etc/nix/nix.conf && echo "sandbox = false" >> /etc/nix/nix.conf
```

Comment:
nitpick: can you split this command into multiple lines like he test? Otherwise it is easy to overlook that this line also does `touch` and `echo`. Mostly because almost all(?) of the rest of the repo2docker code base splits commands into multiple lines connected with `&& \`

Reply:
I did that on purpose, because these three parts absolutely needs to be done together, but sure, I can do that